packages/react/src/ui/skeleton.tsx

```diff
@@ -6,6 +6,7 @@ function Skeleton({
 }: React.HTMLAttributes<HTMLDivElement>) {
   return (
     <div
+      data-testid="skeleton"
```

Comment:
would you consider to use `accesibilityLabel` instead of `data-testId`?

Reply:
as far as I know `accesibilityLabel` is something related to iOS

Reply:
not sure if we need it. I just want data-testid to check if the skeleton is present